.github/workflows/deployProd.yml

```diff
@@ -11,6 +11,17 @@ concurrency:
   group: prod-deploy
 
 jobs:
+  check-stg-release:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./ops
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check that the current release is in the stg environment
+        env:
+            CURL_TIMEOUT: 5
+        run: make check-stg-release check-stg-readiness
```

Comment:
I suspect that I'm just not familiar with how these `jobs` are run, but wouldn't this line create a recursive loop since the job name is also `check-stg-release`?

Reply:
Short answer is no, since `make check-stg-release` is referring to a target with that name in the `Makefile`, not this file, but yeah, maybe we could make the naming convention slightly different to avoid confusion?

Reply:
I added a suggestion for a new name for the job, since there's more value to having that be self-explanatory, and having the makefile targets have a well-defined name structure.